UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -774,7 +781,11 @@ protected virtual IEnumerator LoadBuffer(string sourceUri, GLTF.Schema.Buffer bu
 				{
 					var www = UnityWebRequest.Get(Path.Combine(sourceUri, uri));
 
+#if UNITY_2017_2_OR_NEWER
```

Comment:
@keveleigh pretty sure this was introduced in 2017.1, but I may be wrong.
Please double check.

Reply:
Double checked, and 2017.1 does not have `SendWebRequest`

![image](https://user-images.githubusercontent.com/3580640/36692677-9cfda71c-1aed-11e8-941d-960629a3ee62.png)